source/playbooks/setting-up-playbooks.rst

```diff
@@ -58,17 +58,23 @@ When a run starts
   * **Invite members**: Enable the toggle, and select members who will automatically be invited to the channel when the playbook run starts.
   * **Assign the owner role**: Enable the toggle and select a member. This member is automatically assigned as the owner of the playbook run.
   * **Announce in another channel**: Enable the toggle and select a channel. When the playbook run is started, an announcement is made in the selected channel.
-  * **Send outgoing webhook**: Enable the toggle and enter the webhook you want to use for when the run starts.
+  * **Send outgoing webhook**: Enable the toggle and enter the webhook you want to use for when the run starts. For information about the webhook payload, see the `playbook_run.go file <https://github.com/mattermost/mattermost-plugin-playbooks/blob/master/server/app/playbook_run.go>`_.
```

Comment:
This should point to the root `struct` of the payload, which lives in `playbook_run_service.go`. Also, we can add the latest commit to the URL so that we can highlight the exact lines we mean.So something like
```suggestion
  * **Send outgoing webhook**: Enable the toggle and enter the webhook you want to use for when the run starts. For information about the webhook payload, see the `PlaybookRunWebhookPayload struct <https://github.com/mattermost/mattermost-plugin-playbooks/blob/b4c8058d8660efe35050bc7eb080e3819c7ab09c/server/app/playbook_run_service.go#L176-L185>`_.
```